cmd/stack_package.go

```diff
@@ -102,7 +104,7 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 			appsodyHome := getHome(rootConfig)
 			Debug.Log("appsodyHome is:", appsodyHome)
 
-			devLocal := filepath.Join(appsodyHome, "stacks", "dev.local")
+			devLocal := filepath.Join(appsodyHome, "stacks", imageNamespace)
```

Comment:
I'm concerned about this. I know we had a discussion about whether or not the archives should always be stored in `~/.appsody/stacks/dev.local` or in separate dirs, but I don't think we came to a conclusion. 

With your code here, I believe we would be putting the `dev.local-index.yaml` in `~/.appsody/stacks/imageNamespace`. Therefore there could be multiple `dev.local-index.yaml`s around, and they won't get merged properly.

In my opinion, we should leave this hard coded to "dev.local" _for now._ This seems safest as we continue to develop the new `add-to-repo` command functionality.